README.md

```diff
@@ -1,58 +1,127 @@
-
-## Description
 
-In Portland, housing insecure individuals struggle to maintain documentation, which is often required to receive government and/or non-profit services. With PASS, we are building out an application to enable house insecure individuals to store their personal documents online in decentralized data stores, called Pods. Using [Solid Data Pods](https://solidproject.org/) individuals will have control over which organizations and applications can access their documents. Verified organization will be able to use PASS to request data from an individual and/or add documents (such as references or invoices) to an individuals pod.
+# PASS - Personal Access System for Services
 
-## Terminology
+<img src ="logo.png" width="175">
 
-- Individual - housing insecure person using services to interact with organizations.
+[![License](https://img.shields.io/github/license/codeforpdx/PASS)](https://github.com/codeforpdx/PASS/blob/Master/LICENSE)
+[![Discord](https://img.shields.io/discord/1068260532806766733)](https://discord.gg/Ts923xaUYV)
+[![Pull Requests](https://img.shields.io/github/issues-pr/codeforpdx/PASS)](https://github.com/codeforpdx/PASS/pulls)
+[![Issues](https://img.shields.io/github/issues/codeforpdx/PASS)](https://github.com/codeforpdx/PASS/issues)
+[![Commits](https://img.shields.io/github/commit-activity/m/codeforpdx/PASS)](https://github.com/codeforpdx/PASS/commits/Master)
+[![Build status](https://github.com/codeforpdx/pass/actions/workflows/ci.yml/badge.svg)](https://github.com/codeforpdx/pass/actions?query=workflow%3ABuild)
+![Top Language](https://img.shields.io/github/languages/top/codeforpdx/PASS)
+[![Contributors](https://img.shields.io/github/contributors/codeforpdx/pass)](https://github.com/codeforpdx/PASS/graphs/contributors)
+![Latest Commit](https://img.shields.io/github/last-commit/codeforpdx/PASS/Development)
+
+Welcome! 👋👋🏿👋🏽👋🏻👋🏾👋🏼
+
+PASS is an open source digital wallet for providing home insecure individuals a safe place to store documents within their control. PASS additionally aims to assist caseworkers with processing and providing documents needed to complete the housing assistance application process.
+
+PASS is currently in Development heading towards [Minimum Viable Product](./docs/ROADMAP.md)
+
+## Contents
+
+1.  [Setup Instructions](#1-setup-instructions-🔧)
+2.  [Project Overview](#2-project-overview-🚧)
+       - [Terminology](#terminology-🗣️)
+       - [Features and usage](#features-and-usage-🔀)
+3.  [Contribution Guidelines](#3-contribution-guidelines-🧑‍💻)
+4.  [Contributors](#4-contributors-💻)
+5.  [Tech Stack and Additional Resources](#5-tech-stack-🧪)
+
+---
+## 1. Setup Instructions 🔧
+
+- ### Prerequisites
+
+   Currently we require Node version 16 or higher and NPM for our package manager.  Most places recommend using a node version manager to install node and npm. To proceed using NVM perform the following.. 
```

Comment:
Thanks for adding this. Looks good!

Reply:
Yeah, I agree with Tim. The new updates for our documentation are coming out nicely.